nomad/structs/structs.go

```diff
@@ -8974,6 +8974,7 @@ type ACLPolicy struct {
 	Name        string // Unique name
 	Description string // Human readable
 	Rules       string // HCL or JSON format
+	RulesJSON   *acl.Policy
```

Comment:
I have no particular attachment to this name, it’s awkward, I’m open to suggestions!

Reply:
Maybe `Policies`?

Reply:
It’s just the rules portion though, right? The whole thing is the policy already. I don’t knowwwwwah haha

Reply:
We should also document that this field is generated from Rules on read. It's a bit unusual to have a field only in `nomad/structs/structs.go` and not in its corresponding `api/acl.go` model, so we should let people know why that is.

As far as naming, yeah it's awkward, but I think it fits. I feel like we confuse "ACL", "Rules", "Policies", "Capabilities", etc regularly but as this field is just a specific encoding of another field, we should follow the other field's naming.